flink-cdc-composer/src/main/java/org/apache/flink/cdc/composer/flink/FlinkPipelineComposer.java

```diff
@@ -54,6 +54,11 @@ public class FlinkPipelineComposer implements PipelineComposer {
     private final StreamExecutionEnvironment env;
     private final boolean isBlocking;
 
+    public static FlinkPipelineComposer ofStreamExecutionEnvironment(
+            StreamExecutionEnvironment env) {
+        return new FlinkPipelineComposer(env, false);
+    }
```

Comment:
hmm to me this essentially expose the constructor and it's a bit opaque to set `isBlocking=false` statically here as you can have blocking mode for stream execution env too. My proposal is to make the constructor below public. 
```
  private FlinkPipelineComposer(StreamExecutionEnvironment env, boolean isBlocking) {
        this.env = env;
        this.isBlocking = isBlocking;
    }
```

@PatrickRen I see that you are tagged in the history of the class. Any opinions?